vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/impl/memory/InMemoryImageContent.kt

```diff
@@ -11,4 +12,6 @@ import java.awt.image.BufferedImage
  * @author Luca Rossetto.
  * @version 1.0.0
  */
-data class InMemoryImageContent(override val content: BufferedImage) : ImageContent
+data class InMemoryImageContent(override val content: BufferedImage) : ImageContent {
+    override val id: UUID = UUID.randomUUID()
```

Comment:
I would add this to the constructor. There might be use-cases, where one wants to define this explicitly. You can also use default values in the constructor.